docs/UI-Navigation-Editing-Bar.md

```diff
@@ -0,0 +1,38 @@
+![Navigation Bar Highlighted](images/UIView/NavBarHighlight.png)
+
+## Navigation Bar
+
+The navigation bar is the row of controls at the very top of the window.
+
+* ==:material-database: Data== navigates to the folder that contains the current dataset.
+* ==:material-pipe: Run Pipeline== will launch a pipeline dropdown menu.
+    * **NOTE** Current annotations will be replaced by the pipeline output when it is complete.  You should not perform annotations while a pipeline is running.
```

Comment:
I'm wondering if this note should be referenced in the Pipeline and Training section as well?

Reply:
That kinda feels out of scope for this page, but I will defer to you.